app/views/content/life-as-a-teacher/_categories.html.erb

```diff
@@ -11,6 +11,15 @@
               image: "static/images/content/hero-images/teacher.jpg"
             )) %>
         </ul>
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Teaching as a career",
+              description: "Learn about the career progression in teaching, and the support available for early career teachers and career changers.",
+              path: "/life-as-a-teacher/teaching-as-a-career",
+              image: "static/images/content/hero-images/teacher3.jpg"
```

Comment:
I love this image but we have the same teacher twice (out of three images) on the Life as a teacher page now - should we use a different image instead @jenhadfield-dfe ?

Reply:
updated the image to a different one